frontend/src/metabase-lib/Question.ts

```diff
@@ -1046,13 +1048,12 @@ class Question {
   }
 
   query(metadata = this._metadata): Query {
+    const databaseId = this.legacyQuery()?._databaseId() ?? null;
```

Comment:
This is the only `_database()`/`_databaseId()` usage left outside of old `Query`/`AtomicQuery`/`NativeQuery`/`StructuredQuery`.

Reply:
Maybe `const databaseId = this.datasetQuery().database`?